deploy/operator_ironic.yaml

```diff
@@ -14,6 +14,25 @@ spec:
     spec:
       serviceAccountName: metal3-baremetal-operator
       hostNetwork: true
+      initContainers:
+        - name: static-ip-set
+          image: quay.io/imain/static-ip-manager:latest
```

Comment:
I can help you set up more official builds of any new images.

Reply:
Yeah we need to get the repo and image into official repos.  Thanks Doug! :)

Reply:
I think we need to set that image up under metal3-io on quay before we merge this.